param/cobalt.yaml

```diff
@@ -160,3 +160,14 @@ sensor:
             x: 0
             y: 0
             z: 0
+hydrophones:
+    pinger:
+        depth: -4.9
```

Comment:
Correct. It's used for calculating if a position measurement is feasible. We know our depth. If we know the pingers depth, we can check the resulting depth the pinger claims we are at against what we actually are at. To calculate the depth that the pinger claims we are at, we must know the depth of the pinger.